packages/grafana-ui/src/components/Logs/LogRowMessageParsed.tsx

```diff
@@ -8,16 +8,16 @@ import { getAllFields } from './logParser';
 
```

Comment:
Just a nit (probably invalid):

is the `Parsed` suffix in this component name referring to the same "context" as the renaming in this PR?
if so, does it make sense to also rename the component to avoid confusion?

Reply:
Yeah, good catch. Renamed in https://github.com/grafana/grafana/pull/28881/commits/79317448f2c8f561808f86f5e674c686d1aed91f.